src/components/Reviews.jsx

```diff
@@ -2,19 +2,24 @@ import { useEffect, useState } from "react";
 import { getReviews } from "../api";
 import { CategoriesDropdown } from "./CategoriesDropdown";
 import { Review } from "./Review";
+import { SortByDropdown } from "./SortByDropdown";
 
 export function Reviews() {
   const [reviews, setReviews] = useState([]);
   const [chosenCategory, setChosenCategory] = useState("all");
+  const [chosenSortBy, setChosenSortBy] = useState("votes:desc");
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     setIsLoading(true);
 
```

Comment:
When this useEffect runs, you are rendering everything. I would consider changing it so only the reviews render, and not the search bar. 👍